### PR TITLE
allow null body

### LINKED ISSUE #20: handle no body
Looks like read auth is broken. Can't read a private issue without a body? Perhaps a warning vs a crash makes sense? 

`command-ask/src/helpers/issue-fetching.ts:44:11`

```diff
! Issue body or URL not found
```

_Originally posted by @ubiquity-os in https://github.com/ubiquity/.github-private/issues/6#issuecomment-2440009316_

### PR DESCRIPTION
**Pull request by Keyrxng**
Resolves #20

### CODE REVIEW

**Comment by 0x4007** [comment]
Can you prove it works?

**Comment by Keyrxng** [comment]
> Can you prove it works?

image

**Comment by 0x4007** [comment]
Why don't you share a link so we can just test it? Also it is complaining about the same no body error at the bottom.

**Comment by Keyrxng** [comment]
> Why don't you share a link so we can just test it? Also it is complaining about the same no body error at the bottom.

I made the change performed the QA and then reverted the change and performed it again which is why the error is at the bottom.

> Can't read a private issue without a body? 

The issue as per the spec was based in a private repo. 

Codebase search of that string reveals nothing, so it will not throw on an empty body.
image

File: src/helpers/issue-fetching.ts [hunk 1]
```diff
@@ -41,21 +41,19 @@ export async function fetchLinkedIssues(params: FetchParams) {
...
   const issueKey = createKey(issue.html_url);
   const [owner, repo, issueNumber] = splitKey(issueKey);
-  const linkedIssues: LinkedIssues[] = [{ body: issue.body, comments, issueNumber: parseInt(issueNumber), owner, repo, url: issue.html_url }];
+  const linkedIssues: LinkedIssues[] = [{ body: issue.body || "", comments, issueNumber: parseInt(issueNumber), owner, repo, url: issue.html_url }];
   const specAndBodies: Record<string, string> = {};
   const seen = new Set<string>([issueKey]);
 
   comments.push({
-    body: issue.body,
+    body: issue.body || "",
```
Comment (0x4007) [inline src/helpers/issue-fetching.ts:56]: I would definitely do null

**Review by 0x4007 — APPROVED** [review]

File: src/helpers/issue-fetching.ts (hunk 1 above)
Comment (Keyrxng) [reply src/helpers/issue-fetching.ts:56]: fixed in #28

**Pull request merged**